app/src/main/java/com/duckduckgo/mobile/android/duckduckgo/domain/bookmark/Bookmark.java

```diff
@@ -0,0 +1,15 @@
+package com.duckduckgo.mobile.android.duckduckgo.domain.bookmark;
+
+/**
+ * Created by fgei on 6/12/17.
+ */
+
+public interface Bookmark {
+    String getId();
+
+    String getName();
+
+    String getUrl();
+
+    int getIndex();
```

Comment:
No, keep it, I think it's valid in this case. As I read on, I better saw why you used the approach you did.